### PR TITLE
Fix: Add condition to check the output of "rpm-ostree status" firstly

### PR DESCRIPTION
**Pull request by wushiqinlou**
Signed-off-by: jiazhang <jiazhang@redhat.com>

### All Pull Requests:

Check all that apply:

* [ ] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This enhancement is to add optional condition to check if the vm is from edge image

### CODE REVIEW

File: insights/combiners/rhel_for_edge.py [hunk 1]
```diff
@@ -58,13 +59,20 @@ class RhelForEdge(object):
...
+                self.is_edge = True
+                if units.is_running("rhcd.service"):
+                    self.is_automated = True
+        elif rpms and cmdline and redhatrelease:
+            if ('rpm-ostree' in rpms and 'yum' not in rpms) and \
+                    ('ostree' in cmdline) and \
+                    ("red hat enterprise linux release" in redhatrelease.raw.lower()):
+                self.is_edge = True
+                if units.is_running("rhcd.service"):
+                    self.is_automated = True
```
Comment (xiangce) [inline insights/combiners/rhel_for_edge.py:78]: Better add one more `else` at the end to handle (Skip) whenever any one of the `rpms and cmdline and redhatrelease` is missing.
And test it.

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/combiners/rhel_for_edge.py (hunk 1 above)
Comment (wushiqinlou) [reply insights/combiners/rhel_for_edge.py:79]: @xiangce - I updated code to raise SkipComponent when `rpmostreestatus` or any of the `[rpms` , `cmdline`, redhatrelease]` is missing.

**Review by xiangce — APPROVED** [review]

File: insights/combiners/rhel_for_edge.py [hunk 2]
```diff
@@ -13,9 +13,11 @@
 from insights.parsers.installed_rpms import InstalledRpms
 from insights.parsers.systemd.unitfiles import ListUnits
 from insights.parsers.redhat_release import RedhatRelease
+from insights.parsers.rpm_ostree_status import RpmOstreeStatus
+from insights.parsers import SkipComponent
```
Comment (xiangce) [inline insights/combiners/rhel_for_edge.py:17]: This `SkipComponet` is moved to the `insights.core.exceptions` module, please import it from there.

File: insights/tests/combiners/test_rhel_for_edge.py [hunk 3]
```diff
@@ -1,7 +1,10 @@
+import pytest
+from insights.core.dr import SkipComponent
```
Comment (xiangce) [inline insights/tests/combiners/test_rhel_for_edge.py:2]: Same comment as above.

File: insights/combiners/rhel_for_edge.py (hunk 2 above)
Comment (wushiqinlou) [reply insights/combiners/rhel_for_edge.py]: @xiangce - Updated.

**Review by xiangce — APPROVED** [review]

**Pull request merged**